app/src/main/res/resource/values-de/strings.xml

```diff
@@ -56,20 +56,22 @@
     <string name="unsplash">Unsplash.com</string>
```

Comment:
l. 50: LOL, "relative"? What about "ähnliche Fotos"? (or one of the [thousand other translations](https://dict.leo.org/englisch-deutsch/related)) Same for "Sammlungen" in l. 51.
In any case "relative" is wrong here.